CONTRIBUTING.md

```diff
@@ -365,7 +366,7 @@ about:
 
   - It may be because of a feature freeze due to an upcoming release. During this time,
     only bug fixes are taken into consideration. If your pull request is a new feature,
-    it will not be prioritized until the release is over. Wait for release.
+    it will not be prioritized until the release is out. Wait for the release.
```

Comment:
Nit: you can avoid the sentence-ending preposition: s/the release is out/after the release/.